lib/active_merchant/billing/gateways/cecabank.rb

```diff
@@ -168,7 +168,7 @@ def commit(action, parameters)
           'AcquirerBIN' => options[:acquirer_bin],
           'TerminalID' => options[:terminal_id]
         )
-        url = (test? ? self.test_url : self.live_url) + "/cgi-bin/#{action}"
+        url = (test? ? self.test_url : self.live_url) + "#{action}.action?"
```

Comment:
Minor nitpick, but I think technically the question mark is not part of the url, but rather initiates the query? Maybe we should add it as part of the xml below: `xml = ssl_post("#{url}?", post_data(parameters))` (if that actually works).

Reply:
Good point! I made that update, and all of the tests are still working fine.